debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/processor/memory/MemoryLogMinerEventProcessor.java

```diff
@@ -41,6 +43,7 @@
  *
  * @author Chris Cranford
  */
+// TODO: can this be a caching impl now as well, just with map
```

Comment:
Is this todo still relevant?

Reply:
Yes.  The previous/current impl of MemoryLogMinerEventProcessor does not extend from the new AbstractTransactionCachingLogMinerEventProcessor impl atm.  It seems like it could extend it, but wasn't sure if it would still be better to leave it as is.  I plan on trying it out and running through the test suite to verify.

Reply:
I think its perfectly fine to extend it; I would just suggest doing that part in a separate commit in the same PR would be the only ask, so we sort of distinguish between refactors & ehcache implementation details.